DataFormats/Legacy/HLT/include/AliceHLT/TPCRawCluster.h

```diff
@@ -33,9 +33,8 @@
 #include <fstream>  // ifstream
 #include <cstring>  // memcpy
 
-namespace ALICE {
-namespace HLT {
-namespace TPC {
+namespace o2 {
+namespace AliceHLT {
```

Comment:
Since you are touching this, consider applying the correct rules for namespaces: lower case underscore convention: `alicehlt` or `alice_hlt`